boards/microbit/dist/openocd.cfg

```diff
@@ -0,0 +1,3 @@
+transport select swd
```

Comment:
this file is basically a duplication of `airfy-beacon/dist/openocd.cfg`, `yunjia-nrf51822/dist/openocd.cfg`, and possibly more. Cant we better put it in some shared location for all of them?!

Reply:
> Cant we better put it in some shared location for all of them?!

See #10243, this is a first step.

Reply:
nice one. So would you mind to get #10243 in first, adapt this PR, and then merge this PR? I will actively help to get this all done tomorrow :-)